packages/cli/test/integration/shared/types.ts

```diff
@@ -47,10 +47,13 @@ type EndpointGroup =
 	| 'folder'
 	| 'insights';
 
+type ModuleName = 'insights';
```

Comment:
I guess eventually we'll move all these tests into the module at which point we don't need any of this anymore. Right?